Makefile

```diff
@@ -105,7 +106,7 @@ docker_build_clearlinux:
 	@echo "Building docker image"
 	@echo OpenVINO Model Server version: $(OVMS_VERSION) > version
 	@echo Git commit: `git rev-parse HEAD` >> version
-	@echo OpenVINO version: 2019_R2 clearlinux >> version
+	@echo OpenVINO version: 2020_R1 clearlinux >> version
```

Comment:
that will be the case  shortly but for now it is 2019_R3

Reply:
We should parametrize it anyway.
Also there are some steps repeating between docker builds in Makefile so we could make them in separate target and add as dependency.